zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/service/NewSessionsServiceImpl.java

```diff
@@ -73,67 +72,65 @@ public Session get(HttpServletRequest id) throws IOException {
      */
     @Override
     public Session get(String id) throws IOException {
-        com.session.service.Session clientSession = client.getSessionByID(id);
+        if (StringUtils.isEmpty(id)) {
+            return null;
+        }
 
+        com.github.onsdigital.session.service.Session clientSession = client.getSessionByID(id);
         return createZebedeeSession(clientSession);
     }
 
     /**
-     * Find a {@link Session} assocaited with the user email.
+     * Find a {@link Session} associated with the user email.
      *
      * @param email the user email address to find.
      * @return a {@link Session} for the requested email if it exists and is not expired. Return null otherwise.
      * @throws IOException for any problems getting the session.
      */
     @Override
-    public Session find(String email) throws IOException {
-        com.session.service.Session clientSession = client.getSessionByEmail(email);
-
-        Session session = createZebedeeSession(clientSession);
-        return session;
+    public Session find(String email) throws IOException, SessionClientException {
```

Comment:
Do the same check here

`        if (StringUtils.isEmpty(email)) {
            return null;
        }`

Can  you also update the javadoc to state it will return null if the email is null or empty please.